packages/terra-application-utility/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Unreleased
 
 2.11.0 - (May 21, 2019)
 ------------------
+### Changed
+* Removed all inline styles and implemented the same styles using external css.
```

Comment:
Mention Removed inline style for Dev-site components

Reply:
Update all the 6 CHANGELOG.md files with the suggested change.